web/packages/teleport/src/services/integrations/types.ts

```diff
@@ -137,19 +139,69 @@ export const awsRegionMap = {
 };
 
 export type Regions = keyof typeof awsRegionMap;
-export type IntegrationExecuteRequest = {
+
+// RdsEngines are the expected backend string values,
+// used when requesting lists of rds databases of the
+// specified engine.
+export type RdsEngines =
+  | 'aurora'
+  | 'aurora-mysql'
+  | 'aurora-postgresql'
+  | 'mariadb'
+  | 'mysql'
+  | 'postgres';
+
+// RdsEngineIdentifier are the name of engines
+// used to determine the grouping of similar RdsEngines.
+// eg: if `aurora-mysql` then the grouping of RdsEngines
+// is 'aurora, aurora-mysql`, they are both mysql but
+// refer to different versions. This type is used solely
+// for frontend.
+export type RdsEngineIdentifier =
+  | 'mysql'
+  | 'postgres'
+  | 'aurora-mysql'
+  | 'aurora-postgres';
+
+export type AwsOidcListDatabasesRequest = {
+  // engines is used as a filter to get a list of specified engines only.
+  // The following values are accepted.
+  // mysql related:
+  //   aurora (for MySQL 5.6-compatible Aurora)
+  //   aurora-mysql (for MySQL 5.7-compatible and MySQL 8.0-compatible Aurora)
+  //   mysql
+  //   mariadb
+  // postgres related:
+  //   aurora-postgresql
+  //   postgres
```

Comment:
I'm not sure if we need to document the possible values, isn't it mainly duplication of the type?